ELECTRON.md

```diff
@@ -1,15 +1,29 @@
-## Configure
-
-Electron expects a node (in the branch `feat/thin-wallet`) running with `--status 8080` and `--wallet-index` parameter
-
 ## Install
 
 Must use node v8.9.0
 
+## Running on Electron
```

Comment:
We must update the QA and include a new Ledger section there.

Reply:
Done.